src/metabase/task/send_anonymous_stats.clj

```diff
@@ -0,0 +1,44 @@
+(ns metabase.task.send-anonymous-stats
+  "Contains a Metabase task which periodically sends anonymous usage information to the Metabase team."
+  (:require [clojure.tools.logging :as log]
+            (clojurewerkz.quartzite [jobs :as jobs]
+                                    [triggers :as triggers])
+            [clojurewerkz.quartzite.schedule.cron :as cron]
```

Comment:
I'm sorry some of the other namespaces are done wrong, I will have to fix those. Anyways here you should just do it like

```clojure
(:require [clojure.tools.logging :as log]
          [clojurewerkz.quartzite.jobs :as jobs]
          [clojurewerkz.quartzite.schedule.cron :as cron]
          [clojurewerkz.quartzite.triggers :as triggers])
```